api_tests/config.py

```diff
@@ -22,10 +22,11 @@ def parse():
 
     # Make sure config file has all required keys
     missing_key = verify_keys(config_dict, [
-      'url',
-      'access_token',
-      'snippet',
-      'user_id',
+        'url',
+        'access_token',
+        'snippet',
+        'user_id',
+        'invalid_snippet_key',
```

Comment:
Hard coding is a collision risk. We did not reserve any names when we began the project, so it is up to the developer to select a name that will not collide with a real one in her repository, otherwise the test will fail.